lib/Caxy/HtmlDiff/HtmlDiff.php

```diff
@@ -666,24 +678,14 @@ protected function isTag($item)
         return $this->isOpeningTag($item) || $this->isClosingTag($item);
     }
 
-    /**
-     * @param string $item
-     *
-     * @return bool
-     */
-    protected function isOpeningTag($item)
+    protected function isOpeningTag($item) : bool
     {
-        return preg_match('#<[^>]+>\\s*#iUu', $item);
+        return preg_match('#<[^>]+>\\s*#iUu', $item) === 1;
```

Comment:
Other small fix, this method did not return bool, but `int|false` (same for isClosing)